### PR TITLE
updated seeder, fixed backward resolvers, some modifications to grahp…

### PR DESCRIPTION
**Pull request by guy-ro**
- backward resolvers now tested and working
- changes to models for simplifying requestTypes
- changes to seeder for simplifying requestTypes

### CODE REVIEW

File: server/models/requestTypeModel.ts [hunk 1]
```diff
@@ -35,18 +31,7 @@ const RequestTypeSchema = new Schema({
...
-    },
-    fulfilled: {
-      type: [{ type: Types.ObjectId, ref: 'Request' }],
-      default: []
-    },
-    deleted: {
-      type: [{ type: Types.ObjectId, ref: 'Request' }],
-      default: []
-    }
+    type: [{ type: Types.ObjectId, ref: 'Request' }]
```
Comment (meganniu) [inline server/models/requestTypeModel.ts]: Could we add a default value?
```
default: []
```

**Review by meganniu — APPROVED** [review]
LGTM!! thnx

File: server/models/requestTypeModel.ts (hunk 1 above)
Comment (guy-ro) [reply server/models/requestTypeModel.ts]: done!

**Review by kevin-wangg — APPROVED** [review]
LGTM

**Pull request merged**